reference/pipeline/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: single
+title:  "Spinnaker Pipeline Reference"
+sidebar:
+  nav: reference
+---
+
+* [Pipeline Expression Reference](/reference/pipeline/expressions/)
+* [Pipeline Stages](/reference/pipeline/stages/)
```

Comment:
nit: do we want to say "Pipeline Stages Reference" for consistency here? I don't have strong feelings, but I think it looks a bit funny to have "Pipeline Expression Reference" and then just "Pipeline Stages."

Reply:
I was matching the existing titles on the target pages ;)